dbt/models/ccao/schema.yml

```diff
@@ -50,7 +56,25 @@ sources:
             description: '{{ doc("shared_column_year") }}'
 
       - name: pin_nonlivable
+        description: '{{ doc("table_pin_nonlivable") }}'
         tags:
           - type_condo
 
       - name: pin_weird
+        description: '{{ doc("table_pin_weird") }}'
+
+exposures:
+  - name: commercial_valuation_data
+    label: Commercial Valuation Data
+    type: dashboard
+    url: https://datacatalog.cookcountyil.gov/Property-Taxation/Assessor-Commercial-Valuation-Data/csik-bsws
+    depends_on:
+      - source('ccao', 'commercial_valuation')
+    owner:
+      name: Data Department
+    description: |
+      Commercial valuation data collected and maintained by the Cook County Assessor's Office, from 2021 to present.
+
+      Notes: Refreshed annually, as data is made available by the commercial team.
+
+      Use cases: Can be used to look up the number of units in a multi-unit apartment building.
```

Comment:
I added this exposure since we have it as an Open Data asset. Not sure why it wasn't included before.